src/api.ts

```diff
@@ -0,0 +1,12 @@
+const USERS_URL = 'https://jsonplaceholder.typicode.com/users';
+const TODOS_URL = 'https://jsonplaceholder.typicode.com/todos';
```

Comment:
не уверен что нам нужно две константы

можно обойтись одной, а в нужном месте подставлять доп значения 
```suggestion
const API_URL = 'https://jsonplaceholder.typicode.com/';
```